indra_world/sources/sofia/processor.py

```diff
@@ -107,12 +108,39 @@ def _build_influences(self, rel_dict):
         return stmt_list
 
     def get_event(self, event_entry):
+        """Get an Event with the pre-set grounding mode
+
+        The grounding mode is set at initialization of the class and is
+        stored in the attribute `grounding_mode`.
+
+        Parameters
+        ----------
+        event_entry : dict[str, str]
```

Comment:
would be a great time to introduce some type hints ;)

Reply:
Want me to make a commit with some of these added?

Reply:
If you don't have anything more exciting in mind, sure! But not critical in this case, I think.

Reply:
Done in 569a349

Reply:
I was even able to steal the types @johnbachman wrote in the sklearn/belief PR too ;)

Reply:
Great, thanks!